### PR TITLE
docs: mention header buttons lib

### PR DESCRIPTION
**Pull request by vonovak**
this library was previously mentioned in there but was removed with version 5. I believe it is handy to have it mentioned in there, since I think many people will look for something like this.

shameless plug - I'm the author of the lib

### CODE REVIEW

File: versioned_docs/version-5.x/header-buttons.md [hunk 1]
```diff
@@ -35,7 +35,8 @@ function StackScreen() {
 }
 ```
 
-When we define our button this way, the `this` variable in `options` is _not_ the `HomeScreen` instance, so you can't call `setState` or any instance methods on it. This is pretty important because it's extremely common to want the buttons in your header to interact with the screen that the header belongs to. So, we will look how to do this next.
```
Comment (satya164) [inline versioned_docs/version-5.x/header-buttons.md:38]: Is this removal intentional?

File: versioned_docs/version-5.x/header-buttons.md [hunk 2]
```diff
@@ -35,7 +35,8 @@ function StackScreen() {
...
+> Note that there is a community-developed solution for rendering header buttons that takes care of the correct styling - see [react-navigation-header-buttons](https://github.com/vonovak/react-navigation-header-buttons)
```
Comment (satya164) [inline versioned_docs/version-5.x/header-buttons.md:38]: Can you add the library here? https://reactnavigation.org/docs/community-libraries-and-navigators

**Review by satya164 — CHANGES_REQUESTED** [review]

File: versioned_docs/version-5.x/header-buttons.md (hunk 2 above)
Comment (satya164) [reply versioned_docs/version-5.x/header-buttons.md:38]: On a sidenote, it'll be cool if the header buttons library used the theme from `react-navigation` so that users could customize the colors in one place, also would be good to use the `tintColor` argument in the examples for the buttons.

File: versioned_docs/version-5.x/header-buttons.md (hunk 1 above)
Comment (vonovak) [reply versioned_docs/version-5.x/header-buttons.md:38]: sorry, I put it back

File: versioned_docs/version-5.x/header-buttons.md (hunk 2 above)
Comment (vonovak) [reply versioned_docs/version-5.x/header-buttons.md:40]: @satya164 the library is already in the list

> On a sidenote, it'll be cool if the header buttons library used the theme from react-navigation

yeah, that's a good point, I'll add that

Comment (vonovak) [reply versioned_docs/version-5.x/header-buttons.md:40]: the lib now takes colors from react navigation's theme :)

**Review by brentvatne — APPROVED** [review]
